utils/build_and_install.sh

```diff
@@ -635,7 +635,7 @@ then
   echo_log "================================"
   echo_log "== Setting up ROS environment =="
   echo_log "================================"
-  if [ ! -e /opt/ros/${ROS_DISTRO}/setup.bash ] && $NOT_CLONE_ONLY
```

Comment:
Maybe:
```bash
if [ ! -e /opt/ros/${ROS_DISTRO}/setup.bash ] && $NOT_CLONE_ONLY && $ROS_FROM_APT
```
Since the purpose of this part is to setup the ROS mirror if it is not installed but it shouldn't redo this if I don't happen to have `source /opt...` in my .bashrc/.zshrc